packages/luna/src/components/Dialog/Dialog.tsx

```diff
@@ -1,67 +1,42 @@
-import React, { useCallback, useRef } from "react";
-import { Button, ButtonSize, ButtonType } from "@/components/Button";
-import { X } from "react-feather";
+import * as DialogComponent from "@radix-ui/react-dialog";
```

Comment:
`RadixDialog` is more elegant. 

```suggestion
import * as RadixDailog from "@radix-ui/react-dialog";
```